udata_front/theme/gouvfr/templates/dataset/display.html

```diff
@@ -15,7 +15,7 @@
     'description': dataset.description|mdstrip(60)|forceescape,
     'image': dataset.organization and dataset.organization.logo(external=True) or '',
     'keywords': [_('dataset')] + dataset.tags,
-    'robots': 'noindex, nofollow' if dataset.is_hidden or 'inspire:identifier' in dataset.extras else '',
+    'robots': 'noindex, nofollow' if not dataset.resources or 'inspire:identifier' in dataset.extras else '',
```

Comment:
We should actually keep on using `is_hidden`, where the logic is.

Reply:
If indexable by data.gouv.fr, it should probably be indexable and used for crawling by search engine?

Reply:
Done in f70fb40f